app/services/hyrax/iiif_authorization_service.rb

```diff
@@ -0,0 +1,18 @@
+module Hyrax
+  class IIIFAuthorizationService
+    attr_reader :controller
+    def initialize(controller)
+      @controller = controller
+    end
+
+    def can?(_action, object)
```

Comment:
I'm confused why you are ignoring the passed in action. Please clarify in code comments as this is a surprise for me.

Reply:
@jeremyf I pulled this code over from Hyku: https://github.com/samvera-labs/hyku/blob/master/app/services/iiif_authorization_service.rb#L7-L9

@jcoyne ideas?

Reply:
Looking at the clock (and knowing my day is wrapping up soon), if someone adds a comment to the method clarifying the intention of ignoring the `_action` and favoring the `:show` action for all permissions, then I am okay with someone dismissing my requested change.

Reply:
The intention here was that this is a sample implementation.  We're providing the action variable in case you want to provide a substitute implementation that does something more complicated.

Reply:
Will do! Thanks, @jeremyf.  I will do that in a follow-up PR and tag you.

Reply:
FWIW, RIIIF has an `:info` https://github.com/curationexperts/riiif/blob/6acf20d6ac5101a49b5e7b55afa15cf0be8c91e7/app/controllers/riiif/images_controller.rb#L35 and a `:show` action.